### ISSUE TITLE
ModelExplainer: Sample from all of training data

### ISSUE THREAD

**Issue by torbjorgen** [Clinical-Genomics/rdds#198]
When creating the negative reference data set for model explainer,
the VRS implementation consumes all of RAM due to previous double
shuffle call. The dataset did not fit into node twice,
especially not after a regular training run.

Solve the post-training step RAM situation as well
as properly sampling a large pool of TN for reference
assembly.
